lib/types/types.go

```diff
@@ -212,3 +213,81 @@ func GetDurationValue(v interface{}) (time.Duration, error) {
 		return time.Duration(n) * time.Millisecond, nil
 	}
 }
+
+// HostAddress stores information about IP and port
```

Comment:
As we're at it, I recommend moving this type to a dedicated `net.go` or `ip.go` file instead. What do you think?

Reply:
`net.ip` sounds good to me, but I have no knowledge on the codebase beside this issue 🙇🏻 Your opinions are more important for me, which one we should choose?

Reply:
Moved to `net.go` just, can move later if you wish.

Reply:
Perfect, I would have prefered `net.go` indeed 👍🏻 🙇🏻